tests/unit/test_api_citation_blueprint.py

```diff
@@ -0,0 +1,66 @@
+from flask import Flask
+import pytest
+
+from api.blueprints.drbCitation import citationFetch
+from api.utils import APIUtils
+
+
+class TestCitationBlueprint:
+    @pytest.fixture
+    def mockUtils(self, mocker):
+        return mocker.patch.multiple(
+            APIUtils,
+            formatResponseObject=mocker.DEFAULT,
+            normalizeQueryParams=mocker.DEFAULT
+        )
+
+    @pytest.fixture
+    def testApp(self):
+        flaskApp = Flask('test')
+        flaskApp.config['DB_CLIENT'] = 'testDBClient'
+
+        return flaskApp
+
+    def test_citationFetch_success(self, mockUtils, testApp, mocker):
+            mockDB = mocker.MagicMock()
+            mockDBClient = mocker.patch('api.blueprints.drbCitations.DBClient')
+            mockDBClient.return_value = mockDB
+
+            mockDB.fetchSingleWork.return_value = 'dbWorkRecord'
+
+            mockUtils['formatResponseObject'].return_value\
+                = 'citationResponse'
+
+            with testApp.test_request_context('/'):
```

Comment:
Right now these tests are failing because the don't have a value for the `formats=` param. You can fix that by giving the `test_request_context` method '/?formats=mla' (or whatever makes sense). Same for the second test below